src/AppIdentityCredentials.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/*
+ * Copyright 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Auth;
+
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Client;
+use google\appengine\api\app_identity\AppIdentityService;
```

Comment:
We actually don't, because if we do then it conflicts with the library on appengine (which gets loaded automatically).

I'm not sure how to best handle that... I suppose a comment on this line would be appropriate. Or adding a mock class somewhere.